### PR TITLE
Bump to Scala 2.13.11 and 3.3.0

### PR DESCRIPTION
**Pull request by adpi2**

### CODE REVIEW

File: scalafix/output/src/main/scala/migrate/RscCompat.scala [hunk 1]
```diff
@@ -12,7 +12,7 @@ object RscCompat {
   }
 
   class Patterns {
-    val List() = List.apply[Nothing]()
+    val List() = List()
```
Comment (adpi2) [inline scalafix/output/src/main/scala/migrate/RscCompat.scala:15]: Tried to debug this but it seems the bug comes from semantidb or Scalafix. There is no `TypeApplyTree` here anymore.

File: scalafix/output/src/main/scala/incompat/Cats5.scala [hunk 2]
```diff
@@ -33,7 +33,7 @@ trait MapInstances extends cats.kernel.instances.MapInstances {
             })
           }
           .value
-        G.map[Map[K,B], Map[K,B]](gbb)(_.toMap[K, B](<:<.refl))
+        G.map[Map[K,B], Map[K,B]](gbb)(_.toMap(<:<.refl))
```
Comment (adpi2) [inline scalafix/output/src/main/scala/incompat/Cats5.scala:36]: Same here.

File: migrate/src/main/scala/migrate/internal/InitialLib.scala [hunk 3]
```diff
@@ -162,34 +162,21 @@ object InitialLib {
     Set(
       Organization("com.softwaremill.scalamacrodebug")            -> Name("macros"),
       Organization("com.github.ajozwik")                          -> Name("macro"),
-      Organization("io.argonaut")                                 -> Name("argonaut"),
```
[the remark this answers is not in the document]
Comment (adpi2) [reply migrate/src/main/scala/migrate/internal/InitialLib.scala:165]: > Do new versions of these libraries not use macros anymore?

Yes they do but they also have a Scala 3 version. So the migrate tool will recommend to bump to the Scala 3 available version.

File: migrate/src/test/scala/migrate/MigrateScalacOptionsSuite.scala [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
+      "-encoding",
+      "UTF-8",
+      "-language:higherKinds",
+      "-language:existentials",
+      "-Xlint:_,-type-parameter-shadow",
+      "-Xfatal-warnings",
+      "-Wunused patvars"
+    )
+    val expected = Seq(
+      "-encoding UTF-8",
```
[the remark this answers is not in the document]
Comment (adpi2) [reply migrate/src/test/scala/migrate/MigrateScalacOptionsSuite.scala:19]: We try to keep the separator given as input. To reduce the difference between what's written in the user build and the proposed changes.

File: migrate/src/main/scala/migrate/internal/CommandLineParser.scala [hunk 5]
```diff
@@ -19,7 +19,7 @@ import scala.annotation.tailrec
  *
  * Copied from scala/scala
  */
-private object CommandLineParser {
+private[migrate] object CommandLineParser {
```
[the remark this answers is not in the document]
Comment (adpi2) [reply migrate/src/main/scala/migrate/internal/CommandLineParser.scala:22]: I don't really care

**Review by sjrd — APPROVED** [review]

**Pull request merged**